aries-site/src/pages/components/page.mdx

```diff
@@ -0,0 +1,170 @@
+  // represent the example
+  screenContainer={{scale: 0.85}}
+  width="100%"
+>
+  <PageExample />
+</Example>
+
+## Page and PageContent Working Together
+Page is the outer layout component and provides the desired layout context. 
+Page’s `kind` determines which theme-defined alignment, padding, and min/max-widths 
+are applied to its PageContent children.
+
+As a child of Page, PageContent consumes and adheres to the 
+layout styling provided by its parent Page.
+
+```
+<Page kind="wide">
+  <PageContent>
+
+    /* All contents within me will respect the 
+     * max-width, alignment, and padding specified 
+     * by Page's context. */
+     
+  </PageContent>
+  
+  /* Contents outside of PageContent are not bound by 
+   * Page's context. */
+
+</Page>
+```
+
+<br />
+
+In most cases, all content on a page can be wrapped in a [single PageContent component](#simple). 
+However, designs using [differing background colors](#full-width-background-color) to delineate page 
+sections may accomplish this by applying a PageContent per desired section.
+
+## Example use cases
+
+### Simple
+All content is placed within a single PageContent component.
+
+<Example
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/page/SinglePageContent.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/FilterControls/FilterControls.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/FilterControls/FiltersContext.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/PageHeader/PageHeader.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/data/mockData/trainingDataSets.json",
+  ]}
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  width="100%" 
+  showResponsiveControls={['fullScreen']}
+  screenContainer={{ scale: 0.6 }}
+>
+  <SinglePageContent />
+</Example>
+
+### Full-width background color
+Use cases where an application needs a background to span the entire width of a page section 
+is achieved by setting `background={{ fill: "horizontal", ...additional properties }}` on 
+PageContent.
+
+This treatment is often seen on marketing sites where backgrounds help visually 
+block content sections. This might also be seen in a context such as an app-store catalog where 
+the hero section might have an image spanning.
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/page/PageContentExample.js"
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  width="100%" 
+  showResponsiveControls={['fullScreen']}
+  screenContainer={{ scale: 0.57 }}
+>
+  <PageBackGroundExample />
+</Example>
+
+## Variants 
```

Comment:
any reason the Simple and Background Example are not here under Variants? 
Or are we only explicitly having the three `kinds` under variants?

Reply:
mmmm... good question. In my mind I see a distinction between what I is trying to be communicated in the two sections, but I am having trouble articulating... which is a signal that I need to develop greater clarity in my thought, or that there isn't a justifiable distinction.

Let me attempt:
- **Variants** - I view variants as various flavors of the same thing (e.g. chocolate, vanilla, strawberry are variants of ice creams, or hamburger, fried chicken, chef salad are variants of menu items). So when I look at the "variants" section, I see the purpose of the section as presenting a menu of options I can order from.
- **Example use cases** - My intent for this section was more to try to align to or showcase the possibilities of experiences someone might want to create. Following the food analogy, what type of meal experience would you like to create? A quick lunch? A celebratory meal with friends? A romantic date night? These are the things I am trying to accomplish, experience, feel, etc. We can use our menu of options (variants) to create this experience (implement this use case). We can grab white table cloth, fine dishes, candle-light, farm-to-table ingredients to create a romantic date night; we can grab kind="wide" + background props to create a rich marketing site.

I feel like presenting types of experiences and also presenting a menu of options are both valuable. Now my questions are:
- Was this accomplished? Was it done well?
- How comprehensive should our variants be? Should `background` be part of the variants? I'm leaning towards background is a Box prop and Box should do that work, whereas `kind` is specific to page.

End of my rambling thoughts for now...

Reply:
It's a good question -- I think initially we tried to have "Variants" be prop related, then in some cases (templates especially) it's more pattern focused. Maybe something for us to revisit in future. I am okay with the current differentiation of variants/use cases